IDEAS/Examples/PPD12/Structure.mo

```diff
@@ -74,6 +74,8 @@ model Structure "Ppd 12 example model"
     inc=IDEAS.Types.Tilt.Floor) "Floor between hallway and bedroom 1"
     annotation (Placement(transformation(extent={{70,68},{80,88}})));
   IDEAS.Buildings.Components.Window winBed3(
+    hVertical=1.2*sin(winBed3.inc),
```

Comment:
Update revision notes for PPD12 model?

Reply:
See commit e25590cf60a1ef31c4d15274f30aa798c6c6479e van @kldjonge :) Klaas can you add the revision note then?

Reply:
Done (on my forked branch)